http/src/main/resources/reference.conf

```diff
@@ -17,39 +17,40 @@ dataproc {
     "https://www.googleapis.com/auth/userinfo.profile",
     "https://www.googleapis.com/auth/cloud.useraccounts.readonly",
     "https://www.googleapis.com/auth/bigquery"
-  ],
-  # TODO: temp 2019-07-08
-  welderDisabledNotebooksDir = "/home/jupyter-user"
-  welderEnabledNotebooksDir = "/home/jupyter-user/notebooks"
+  ]
   # Unset to use project-level defaults. Note a VPC/subnet with the literal name "default" may not exist.
   #vpcNetwork = "default"
   #vpcSubnet = "default"
   # This image is used for legacy jupyter image where hail is compatible with earlier version of spark
   legacyCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-2-79-debian9-251b76c"
   customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-4-15-debian9-251b76c"
 
+  dataprocReservedMemory = 4.5g
+}
+
+image {
+  welderDockerImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:60e28bc"
```

Comment:
I just realize this refactoring seems to break the regex in https://github.com/broadinstitute/dsp-jenkins/blob/master/jobs/fc-jenkins/WelderUpdateHash.groovy#L32...it was probably broken from a previous PR...

Reply:
Oh, I didn't realize that existed

Reply:
I was testing the regex and it _seems_ to work -- the line `welderDockerImage = …` is unchanged, it's just in a different structure.